app/components/export-basic-metadata.js

```diff
@@ -22,7 +22,7 @@ export default Component.extend({
           "resourceType": doi.types.resourceType,
           "title": doi.title,
           "author": formatCreators(doi.creators, {}),
-          "publisher": doi.publisher,
+          "publisher": doi.publisher.name,
```

Comment:
Is it guaranteed that all Publisher objects will be a hash by the time this goes live, or do we need code that can handle both a hash and the old style string (similar to the approach used by @svogt0511 and @codycooperross for Lupo)?

Reply:
@digitaldogsbody yes that is correct, by the time we go to prod we will have hash for publisher